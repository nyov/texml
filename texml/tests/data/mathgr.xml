<TeXML>Text: |a+b|>2
<math>|a+b|>3</math>
<dmath>|a+b|>4</dmath></TeXML>
