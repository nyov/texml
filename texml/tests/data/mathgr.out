Text: \textvert{}a+b\textvert{}\textgreater{}2 $|a+b|>3$
$$|a+b|>4$$ $$\sqrt{2}$$
